embassy-stm32/src/can/fd/message_ram/enums.rs

```diff
@@ -0,0 +1,233 @@
+// Note: This file is copied and modified from fdcan crate by Richard Meadows
+
+/// Datalength is the message length generalised over
+/// the Standard (Classic) and FDCAN message types
+
+#[derive(Clone, Copy, Debug, PartialEq)]
+pub enum DataLength {
+    Standard(u8),
+    Fdcan(u8),
```

Comment:
If this is part of public interface consider maybe using Extended instead of Fdcan? It's probably related to how fdcan crate named it though.

Reply:
Yes, though I think Classic. 
because it could be either Extended  Standard ID's,
Seems we are using Classic for any CAN 2.0B packets i.e. not Fd

Reply:
Pushed.